templates/hpe/index.html

```diff
@@ -38,10 +38,12 @@ <h1><strong>Hewlett Packard Enterprise and Canonical</strong></h1>
     <hr class="p-rule">
   </div>
   <div class="row">
-    <div class="col-6 col-medium-3">
-      <h2>A single point of contact for technical expertise</h2>
+    <div class="col-6 col-medium-6">
+      <div class="p-block">
```

Comment:
`p-block` is a replacement for `row` isn't it? Why would you use that inside a col?

Reply:
It just adds unnecessary space between the heading and text on small screens.
![image](https://github.com/canonical/ubuntu.com/assets/1413534/fb370224-12db-4f67-9ddb-8a0230e9823e)

Reply:
no, p-block is a shallow strip with padding only at the bottom, see here https://vanillaframework.io/docs/patterns/section

We have a wall of text and the heading is "swallowed" by it, a bit of white space helps make the whole block more readable in my opinion.

Reply:
I created a div... I was trying to avoid creating a useless, but let's keep consistency.